<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestionar Usuario</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>531</width>
     <height>511</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Registrar</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>NOMBRE DE USUARIO:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CONTRASEÑA:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_nombre">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_contra">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>FECHA DE INICIO:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="r_inicio">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_registrar">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>430</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>REGISTRAR USUARIO</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_telefono">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TELEFONO:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>SUPER USUARIO:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="super_no">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>NO</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="super_si">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>170</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>SI</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>421</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>FUNCIONES EXTRA VENDEDOR</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="r_manual">
         <property name="text">
          <string>Crear orden manual</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="r_informes">
         <property name="text">
          <string>Generar informes</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>421</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>FUNCIONES PERSONAL ÁREA</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="r_dim">
         <property name="text">
          <string>Dimensionado</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="r_elab">
         <property name="text">
          <string>Elaboración</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="r_carp">
         <property name="text">
          <string>Carpinteria</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="r_pall">
         <property name="text">
          <string>Pallets</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>491</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>TIPO USUARIO</string>
     </property>
     <widget class="QRadioButton" name="r_vendedor">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>VENDEDOR</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="r_personal">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>PERSONAL DE ÁREA</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="r_porteria">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>PERSONAL DE PORTERÍA</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>NOMBRE COMPLETO:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_fullnom1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Modificar</string>
    </attribute>
    <widget class="QComboBox" name="box_usuario1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>271</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>USUARIO:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_actualizar">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>440</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ACTUALIZAR USUARIO</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QDateEdit" name="m_inicio">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_obtener">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>0</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Obtener datos</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="m_nombre">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="m_contra">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>NUEVO NOMBRE DE USUARIO:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>NUEVA CONTRASEÑA:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>NUEVA FECHA DE INICIO:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>NUEVO TELEFONO:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="m_telefono">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>SUPER USUARIO:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="super_no_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>200</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>NO</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="super_si_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>200</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>SI</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>421</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>TIPO USUARIO</string>
     </property>
     <widget class="QRadioButton" name="r_vendedor_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>VENDEDOR</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="r_personal_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>PERSONAL DE ÁREA</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>421</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>FUNCIONES EXTRA VENDEDOR</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="r_manual_2">
         <property name="text">
          <string>Crear orden manual</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="r_informes_2">
         <property name="text">
          <string>Generar informes</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>421</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>FUNCIONES PERSONAL ÁREA</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="r_dim_2">
         <property name="text">
          <string>Dimensionado</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="r_elab_2">
         <property name="text">
          <string>Elaboración</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="r_carp_2">
         <property name="text">
          <string>Carpinteria</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="r_pall_2">
         <property name="text">
          <string>Pallets</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>NUEVO NOMBRE COMPLETO:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_fullnom2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Retirar</string>
    </attribute>
    <widget class="QComboBox" name="box_usuario2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>271</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>USUARIO:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_retirar">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>RETIRAR USUARIO</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Fecha de retiro:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="r_termino">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>(Indica el ultimo dia de trabajo del dimensionador)</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_atras">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>540</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Atrás</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
